src/stripeTerminal.ts

```diff
@@ -25,17 +28,22 @@ export class StripeTerminal {
     command: SupportedStripeCommand,
     args: Array<string> = [],
   ): Promise<void> {
+    const cliPath = await this.stripeClient.getCLIPath();
+    if (!cliPath) {
```

Comment:
I was gonna suggest we prompt the user if this happens but it looks like we already do that from getCLIPath() -> detectInstalled()!